docs/adguard-for-android/solving-problems/low-level-settings.md

```diff
@@ -76,29 +76,31 @@ If enabled, AdGuard strips Encrypted Client Hello parameters from DNS responses.
 
 #### Ignore unavailable outbound proxy
 
-Enable this feature to make AdGuard send DNS requests directly if the outbound proxy is unavailable.
+If this setting is enabled, AdGuard will send DNS requests directly when the outbound proxy is unavailable.
 
 #### Try HTTP/3 for DNS-over-HTTPS upstreams
 
-By default, all DNS requests for DNS-over-HTTPS are sent via HTTP/2 protocol. If enabled, AdGuard uses HTTP/3 to speed up DNS query resolution for DoH upstreams.
+If this setting is enabled, AdGuard will use HTTP/3 to speed up DNS query resolution for DoH upstreams. Otherwise, AdGuard will revert to its default behavior and use HTTP/2 to send all DNS requests for DNS-over-HTTPS.
 
 #### SERVFAIL failure response
 
-Once enabled, AdGuard sends a SERVFAIL response to the client if all upstreams, including fallback ones, fail to reply. When this setting is disabled, no response is sent to the client.
+If this setting is enabled, AdGuard will send a SERVFAIL response to the client if all upstreams, including fallback ones, fail to reply. When this setting is disabled, no response is sent to the client.
```

Comment:
давай тут тоже с otherwise

Reply:
Можно так
If this setting is enabled and all upstreams, including fallback ones, fail to respond, AdGuard will send a SERVFAIL response to the client. 
Тогда можно и без второго предложения Otherwise, no responses [are | will be] sent. Вроде логика И проходит.